src/Sylius/Bundle/CoreBundle/Resources/config/doctrine/model/Order.orm.xml

```diff
@@ -73,6 +73,10 @@
             </cascade>
         </many-to-many>
 
+        <unique-constraints>
+            <unique-constraint columns="token_value" />
+        </unique-constraints>
+
         <field type="string" name="tokenValue" column="token_value" nullable="true" />
```

Comment:
Unique constraints XML can be replaced by adding this property only.

```suggestion
        <field type="string" name="tokenValue" column="token_value" nullable="true" unique="true" />
```

Reply:
Right, we can simplify it 👌